mods/default/craftitems.lua

```diff
@@ -112,6 +112,16 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 		end
 		data.description = "\""..short_title.."\" by "..data.owner
 		data.text = fields.text:sub(1, max_text_size)
+		while(true) do
+			local num_start,num_end = data.text:find("\r")
+			if num_start ~= nil and num_end ~= nil then
```

Comment:
`if num_start and num_end then`
> Don't compare values explicitly to true, false, or nil, unless it's really needed.

http://dev.minetest.net/Lua_code_style_guidelines#Misc point 3